core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -133,7 +133,8 @@ public static class SnapshotLogEntry implements HistoryEntry {
     private final long timestampMillis;
     private final long snapshotId;
 
-    SnapshotLogEntry(long timestampMillis, long snapshotId) {
+    // VisibleForTesting
+    public SnapshotLogEntry(long timestampMillis, long snapshotId) {
```

Comment:
This is one I could make public via reflection, as the class itself is `public static`.

Reply:
Let's just omit this from the validation.